index.html

```diff
@@ -104,8 +104,7 @@ <h2>Implementations</h2>
             wrapper around Rust, focused on the ATmosphere.
           </li>
           <li>
-            <a href="https://github.com/ipld/libipld">libipld</a> (Rust): fast Rust implementation, extracted
-            from the original <code>ipfs-rust</code> project.
+            <a href="https://github.com/ipld/rust-ipld-core">ipld-cored</a> (Rust): fast Rust implementation.
```

Comment:
Is the d t the end of `ipld-cored` intentional or a typo?

Reply:
Typo. Thanks!